corehq/apps/userreports/reports/view.py

```diff
@@ -369,6 +375,11 @@ def pop_report_builder_context_data(self):
 
     @property
     def saved_report_context_data(self):
+        """
+        Update context with saved report data
+        Override filter_context with saved report config filters
+        :return: updated context
```

Comment:
nit: this doesn't actually update the context - it returns some context that will be inserted into the main context.

Reply:
Fair point